clang/test/AST/Interp/new-delete.cpp

```diff
@@ -245,7 +245,7 @@ namespace std {
 namespace PlacementNew {
   constexpr int foo() { // both-error {{never produces a constant expression}}
     char c[sizeof(int)];
-    new (c) int{12}; // ref-note {{call to placement 'operator new'}} \
+    new (c) int{12}; // ref-note {{this placement new expression is not yet supported in constant expressions}} \
```

Comment:
not C++26

Reply:
Ohhhhhhhhh yeah, it would be better if we could have a more clear diagnostic as the current wording sounds like "we'll get around to it someday" and not "because the language mode isn't new enough".